src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs

```diff
@@ -3079,6 +3089,7 @@ internal enum FileType
             );
 
             [DllImport(PinvokeDllNames.ScrollConsoleScreenBufferDllName, SetLastError = true, CharSet = CharSet.Unicode)]
+            [return: MarshalAs(UnmanagedType.Bool)]
```

Comment:
I understand adding these attributes to be consistent throughout our codebase, but why are they needed?  It seems like 'marshal to bool' is implied...

Reply:
Staffan Gustafsson fixed a bug in one place (there is BOOLEAN not BOOL type in native code). I reviewed all such signatures and added the attribute to ensure that all right.